lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -106,4 +98,23 @@ protected function doSave($id, $data, $lifeTime = 0)
 
         return $this->writeFile($filename, $code);
     }
+
+    /**
+     * @param string $id
+     *
+     * @return array|false
+     */
+    private function includeFileForId($id)
+    {
+        $fileName = $this->getFilename($id);
+
+        // note: error suppression is still faster than `file_exists`, `is_file` and `is_readable`
+        $value = @include $fileName;
```

Comment:
Since this pull requests, a php warning is triggered each time we create a new file cache.

Sample of message:

```bash
include(): Failed opening '/.../app/cache/prod/store_cache/d9/ec/a2/...[sample].storeschema.php
```
Removing the warnings with the php.ini file is an option, but can this Cache be configurable to enable or disable this error suppression?